electron_app/helpers/enviroment.js

```diff
@@ -13,6 +13,8 @@ const constants ={
   IPC_UPLOAD_STATUS: 'upload-status',
   IPC_UPLOAD_CANCEL: 'upload-cancel',
   IPC_DOWNLOAD_CANCEL: 'download-cancel',
+  IPC_GOOGLE_LOGOUT: 'logout-google',
```

Comment:
Above we have `google-login`, maybe this should be `google-logout`? Or is it more like `google-revoke-auth`?